component.json

```diff
@@ -12,6 +12,11 @@
                     "label": "WebHook URL",
                     "required": false
                 },
+                "hmac_key": {
+                    "viewClass": "OptionalRandomString",
```

Comment:
it would be custom view in frontend with empty text field and a button to generate random string into this text field, so if user clicks "Generate" button - it would be populated with random string(or uuid) and will become hmac_key for this webhook